README.md

```diff
@@ -25,24 +32,24 @@ that can be extended beyond just a single group or digitizer.
 For more information about the original project you can view the read me at the 
 [original project](https://github.com/pixie16/paass).   
 
-# Dependencies
-## For all components 
+## Dependencies
+### For all components 
 * ncurses
 * gcc v4.4.6+
 * cmake v2.8.8+
 * [ROOT](https://root.cern.ch)
```

Comment:
Do we have a version requirement for ROOT? Are we still ROOT 5 compatible?

Reply:
We are still ROOT5 compatible. That's the only Version we can easily test with in the CI server.

Reply:
Because it is in the package manager for the current travis os?

Reply:
That is correct.